sources/packages/web/src/services/AuthService.ts

```diff
@@ -25,10 +25,10 @@ export default async function(
     });
     if (keycloak.authenticated) {
       switch (type) {
-        case "student":
+        case ClientIdType.STUDENT:
           store.dispatch("student/setStudentProfileData", keycloak);
           break;
-        case "institute":
+        case ClientIdType.INSTITUTE:
```

Comment:
We should rename INSTITUTE to INSTITUTION in all places as needed.

Reply:
I will do it in part 2